x-pack/auditbeat/processors/add_session_metadata/pkg/processdb/db.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package processdb
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/types"
+)
+
+type DB interface {
```

Comment:
This has only one implementation; do we need an interface here?

Reply:
No, it's not needed. I've removed the interface